.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug report
+description: File a bug report
+labels: ["bug"]
+
+body:
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce the bug.
+      value: |
+        1.
+        2.
+        3.
+        4.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: What should have happened.
+      placeholder: "Example: The validation should have caught the issue."
```

Comment:
These placeholders should be relevant to Severn. What that currently looks like I'm not sure. Maybe for this: "Example: The specified packages should have been installed.", and for the other below, "Example: I received the following error: ...".